Remove duplicated class definition of Conformer and update some new results of Aishell1 and Switchboard.

pengchengguo opened the pull request:
Hi all,

I removed the duplicated class definition of Conformer (Conv2dSubsampling) and updated some new results of Conformer with SpecAugment on Aishell1 and Switchboard corpus.

sw005320 commented:
Very cool!
The results are also amazing!
I have two requests:
- Can you upload your conformer aishell and swbe models? I think you'd have access to upload a model, but if not, please ask me.
- Please make these configs as default (switch the symlink of `train.yaml` and others in these recipes from transformer to conformer one).

@hirofumi0810 and @b-flo, could you review this PR?

pengchengguo commented:
Ok, I will do it now.

b-flo commented on espnet/nets/pytorch_backend/conformer/activation.py:
```diff
@@ -0,0 +1,31 @@
+#                Northwestern Polytechnical University (Pengcheng Guo)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+"""Activation funcitons for Conformer."""
```
```suggestion
"""Activation functions for Conformer."""
```

b-flo reviewed (commented):
LGTM! Maybe `conformer/get_activation.py` should be renamed `conformer/swish.py` and `get_activation(...) ` method moved to `nets_utils` as the function is also for transformer pointwise ff. If you see any use, `transformer-encoder-activation-type` could be split to `transformer-enc-conv-mod-activation-type` and `transformer-enc-pointwise-activation-type` or something like that

sw005320 commented:
@pengchengguo, could you update the master and also remove the conflict?

sw005320 commented on espnet/nets/pytorch_backend/conformer/activation.py:
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright 2020 Johns Hopkins University (Shinji Watanabe)
```
You may not have to put my name ;)
It’s your work!

pengchengguo commented:
Hi Shinji,
I fixed the conflicts and also moved `get_activation()` to `nets_utils.py` as @b-flo 's suggestion. I didn't change the parameter name `transformer-encoder-activation-type` as @b-flo 's suggestion, because the model I uploaded to google drive was trained with `transformer-encoder-activation-type`.

pengchengguo commented:
I found Naoyuki Kamo has made a PR about applying the new version black. But for me, this changes will be failed in CI test.

kamo-naoyuki commented:
You need to apply new black about your updating, of course.

pengchengguo commented:
Hi Naoyuki Kamo, 
Thanks so much for your help. I just found I didn't update the black version for my local environment. That's why it will reformat all files when I run ci/test_python.sh in local.

sw005320 commented:
Many thanks, @pengchengguo!
